smart_contracts/include/sophiatx/smart_contracts/db_resource.h

```diff
@@ -0,0 +1,33 @@
+#ifndef SOPHIATX_DB_RESOURCE_H
+#define SOPHIATX_DB_RESOURCE_H
```

Comment:
Why not use #pragma once instead?

Reply:
"even though #pragma once is widely supported, it is not part of standard." And i kind of like GUARDS more anyway... I dont think there is much difference